### PR TITLE
Non blocking/tester bugfixes

### PR DESCRIPTION
**Pull request by wangmingliang-ms**

### CODE REVIEW

File: Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/task/AzureTaskManager.java [hunk 1]
```diff
@@ -45,107 +45,211 @@ public static AzureTaskManager getInstance() {
         return AzureTaskManager.instance;
     }
 
-    public final Observable<Void> read(Runnable task) {
-        return this.read(new AzureTask<>(task));
+    public final void read(Runnable task) {
```
Comment (Flanker32) [inline Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/task/AzureTaskManager.java:48]: We have many method overloads here, which could be a bit confused, maybe we could leverage patterns like builder to reduce the overloads later

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/webapp/action/CreateWebAppAction.java [hunk 2]
```diff
@@ -146,7 +146,7 @@ private void deploy(final WebApp webapp, final Path application, final Project p
     private void refreshAzureExplorer(WebApp app) {
         AzureTaskManager.getInstance().runLater(() -> {
             if (AzureUIRefreshCore.listeners != null) {
-                AzureUIRefreshCore.execute(new AzureUIRefreshEvent(AzureUIRefreshEvent.EventType.REFRESH, app));
+                AzureUIRefreshCore.execute(new AzureUIRefreshEvent(AzureUIRefreshEvent.EventType.REFRESH, null));
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/webapp/action/CreateWebAppAction.java:149]: We may need to refactor `createListener` in `WebAppModule` for this issue

File: Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/task/AzureTask.java [hunk 3]
```diff
@@ -98,6 +98,10 @@ public AzureTask(Object project, String title, boolean cancellable, Supplier<T>
         this.modality = modality;
     }
 
+    public String getId() {
+        return Integer.toHexString(System.identityHashCode(this));
```
Comment (Flanker32) [inline Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/task/AzureTask.java:102]: Move to a common `Utils`, as same implementation is called in multi codes

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/webapp/action/CreateWebAppAction.java (hunk 2 above)
Comment (wangmingliang-ms) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/webapp/action/CreateWebAppAction.java:147]: let's do it in the future.

**Review by Flanker32 — APPROVED** [review]
LGTM

**Pull request merged**